zh_cn.json

```diff
@@ -0,0 +1,31 @@
+  "lang": {
```

Comment:
Please remove the `"lang":` part, so the json is valid & follows the other files. Otherwise looks good.